CHANGELOG.md

```diff
@@ -1,5 +1,52 @@
 # [Versions](https://mui.com/versions/)
 
+## 6.0.0
+
+<!-- generated comparing v6.0.0-rc.0..master -->
+
+_Aug 27, 2024_
+
+A big thanks to the 14 contributors who made this release possible.
```

Comment:
```suggestion
```

Let's drop this, I would make this changelog entry more of an announcement :)

Reply:
Yep, I just removed it.